calico/networking/configuring/add-maglev-load-balancing.mdx

```diff
@@ -0,0 +1,34 @@
+---
+description: Add Maglev load balancing to a Kubernetes service.
+---
+
+# Add Maglev load balancing to a service
+
+This guide shows you how to enable Maglev-style load balancing to a service.
+Maglev's consistent-hashing load balancing algorithm minimizes the amount of dropped packets and improves perfomance.
+This is especially useful for maintaining sessions during a load balancer failover.
+
+## Prerequisites
+
+* Your Kubernetes cluster uses $[prodname] for networking.
+* All your nodes are running on Linux
+* $[prodname] is configured to use the eBPF data plane.
```

Comment:
The next line implies this one. If you want to, say you need to use calico in ebpf mobe and make a reference to how to enable ebpf

Reply:
You're right, of course. The idea here is to clarify the state of things right here, without requiring people (with Windows nodes) to ferret out that information on another page.

Reply:
The issue with spelling out everything here is that when things change we need to change it in many places. I honestly think itis enough to say, that this feature is only supported with calico ebpf. Those who will get here either know what they are looking for or they can pause, follow a link to "calico ebpf"